client/searchResults.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import NavBar from './navBar';
+import Loader from './loader';
+export default class SearchResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+
+    fetch(`/api/search/${this.props.cityName}`)
+      .then(response => {
+        return response.json();
+      })
+      .then(data => {
+        const cityId = data;
+        fetch(`/api/search/list/${cityId}/${this.state.ratingFilter}`).then(response => response.json()).then(data => {
+          const hotelList = data;
+          this.setState({ hotelList: hotelList, searchButtonClicked: 'hidden', isLoading: false, ratingFilter: this.state.ratingFilter });
+        })
+          .catch(err => console.error(err));
+      })
+      .catch(err => {
+        console.error(err);
+      });
+  }
+
+  render() {
+    if (this.state.isLoading) return <Loader />;
+    let displayedList = null;
+    const hotelList = this.state.hotelList;
+
+    displayedList = hotelList.map((hotel, idx) => {
```

Comment:
You can just do `const displayedList = hotelList.map etc etc`. No need for the separate variable on line 33